src/DataStructures.jl

```diff
@@ -72,6 +72,145 @@ function Base.get!(
     return get!(default, cache.cache, key)
 end
 
+"""
+    Base.get(cache::LRUCache{K, V}, key::K, default::V)
+
+Get the value associated with `key` in `cache`. If the key is not in the
+cache, return the `default`. If the `key` is in `cache`, update the key's priority.
+"""
+function Base.get(cache::LRUCache{K, V}, key::K, default::V) where {K, V}
+    if haskey(cache, key)
+        _update_priority!(cache, key)
+    end
```

Comment:
A common way of writing stuff like this is
```suggestion
    haskey(cache, key) && _update_priority!(cache, key)
```